lib/protocon/plugins/driver_tcp.py

```diff
@@ -93,9 +97,16 @@ def open(self):
 
 		tcp_sock = socket.socket(self._addrinfo.family, self._addrinfo.type)
 		if self.settings['type'] == 'client':
+			if self.url.scheme.startswith('ssl'):
+				context = ssl.create_default_context()
```

Comment:
I think I'm leaving this default context creation alone, since it sets some reasonable defaults to work off in the future.